edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/logs/LogsUploader.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Agent.Core.Logs
```

Comment:
Should this file be named `ILogsUploader`?

Reply:
ah yes